### ISSUE TITLE
Could we improve Log Explorer output in 5.x

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#1423]
Could we apply some colour coding to the messages that appear in the Log Explorer in 5.x? (A bit like in Mantid)

For example:
- INFO: black
- WARNING: orange
- ERROR: red

At the moment it is easy to overlook what appears in the Log Explorer.

**Comment by rozyczko** [comment]
Good idea, we were thinking about doing it as well. However, as an enhancement request, I'd prefer to do it post 5.0.1, please.
